platform/platform-app/src/main/resources/features.xml

```diff
@@ -52,6 +52,13 @@
             <!--mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jaxb-impl/2.2.6_1-->
         <!--</bundle>-->
     <!--</feature>-->
+    <feature name='platform-email' install="automatic" version='${project.version}' start-level='50'>
+        <bundle dependency="true">mvn:org.apache.commons/commons-lang3/${commons-lang3.version}</bundle>
```

Comment:
Do we need lang3? I don't see any usages that aren't also available in 2.6 which is already available in the container.